src/Pages/saranglee/Main/Main.js

```diff
@@ -0,0 +1,90 @@
+import React, { Component } from 'react';
+import { withRouter } from 'react-router-dom';
+
+import Nav from './Components/Nav/Nav';
+import Feed from './Components/Feed/Feed';
+import Story from './Components/Story/Story';
+import Recommendation from './Components/Recommendation/Recommendation';
+
+import '../../../Styles/reset.scss';
+import '../../../Styles/common.scss';
+import '../common.scss';
+import './Main.scss';
+
+class MainSarang extends Component {
+  constructor() {
+    super();
+    this.state = {
+      feedData: [],
+    };
+  }
+
+  componentDidMount() {
+    fetch('http://localhost:3000/data_saranglee/feedData.json', {
+      method: 'GET'
+    })
+    .then(res => res.json())
+    .then(res => {
+      this.setState({
+        feedData: res
+      })
+    })
+  }
+
+  render() {
+    const { feedData } = this.state;
+
+    return (
+      <div className="main">
+        <Nav />
+        <main>
+          <div className="feeds">
+            {feedData.map(feed => {
+              return <Feed data={feed} />;
+            })} 
+          </div>
```

Comment:
```suggestion
          <div className="feeds">
            {feedData.map((feed, idx) => <Feed data={feed} key={idx} />)} 
          </div>
```
- 화살표 함수는 가능한 경우 한줄로 쓰는 것이 좋아보입니다.
- key prop 누락되어 있습니다.

Reply:
넵! 앞으로 한 줄로 가능한 부분은 수정해주신 대로 해놓겠습니다. key props도 꼭 수정해놓을게요. 감사합니다!!